src/main/java/com/iota/iri/service/dto/GetNodeInfoResponse.java

```diff
@@ -18,6 +18,8 @@
     private String latestSolidSubtangleMilestone;
     private int latestSolidSubtangleMilestoneIndex;
 
+		private int milestoneStartIndex;
```

Comment:
I believe this is an unexpected indent.

Reply:
Thanks. I think it's because I used `tab` instead of 4 spaces like the rest of the code used